probe/endpoint/ebpf.go

```diff
@@ -40,11 +43,49 @@ type EbpfTracker struct {
 	closedConnections []ebpfConnection
 }
 
+var releaseRegex = regexp.MustCompile(`^(\d+)\.(\d+).*$`)
+
+func isKernelSupported() error {
+	release, _, err := host.GetKernelReleaseAndVersion()
+	if err != nil {
+		return err
+	}
+
+	releaseParts := releaseRegex.FindStringSubmatch(release)
```

Comment:
You can use `SScanf` see https://github.com/weaveworks/scope/blob/master/probe/endpoint/procspy/proc_linux.go#L49-L61 , which makes the code much simpler.

Extra points for refactoring the code and making `proc_linux.go` use `GetKernelReleaseAndVersion()` (which should probably be moved to a common package).